helper/schema/schema_test.go

```diff
@@ -1886,6 +1889,60 @@ func TestSchemaMap_Diff(t *testing.T) {
 
 			Err: false,
 		},
+
+		// #47 - https://github.com/hashicorp/terraform/issues/824
```

Comment:
What does "47" mean here? I like linking to the issue for more context, but it would be great if there was a short sentence explaining it in case I'm on a plane/without Internet.